macros/sql/weekdays_between_dates.sql

```diff
@@ -0,0 +1,31 @@
+--code source: https://discourse.looker.com/t/how-to-count-only-weekdays-between-two-dates/3345
+{%- macro weekdays_between_dates(start, stop) -%}
```

Comment:
The calculation includes the start date, but not the stop date.  For example:

- If the start date is a Friday and the stop date is the immediately following Saturday, this returns 1.
- If the start date is a Sunday and the stop date is the immediately following Monday, this returns 0.

Is that the expected behavior?

Reply:
Yes this is expected behavior, I added a comment at the top of the file so the user doesn't need to deduce this.